<launch>
    <node pkg="vicon_hand" type="vicon_interp_node" name="vicon_to_hand" output="screen">
            <rosparam param="topicList"> ["/pointer/local_odom","/pinky/local_odom"] </rosparam>
    </node>
</launch>
